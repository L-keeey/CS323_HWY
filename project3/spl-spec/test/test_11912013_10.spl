struct Man
{
  int name;
  int height;
};
int compare1(int x, int y)
{
  if (x > y)
    return 1;
  else if (y > x) 
    return -1;
  else {
    return 0;
  }
}
int test_2_r11(int i, int m, int n)
{
  struct Man man;
  if (i == 0)
    return compare1(m, n);
  else
  {
    return man(m, n);
  }
}