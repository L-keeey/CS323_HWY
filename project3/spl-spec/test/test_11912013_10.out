Error type 11 at Line 23: non-function variable could not apply function invocation.
